docs/platforms/apple/common/configuration/options.mdx

```diff
@@ -271,3 +271,15 @@ String entries do not have to be full matches, meaning the URL of a request is m
 If <PlatformIdentifier name="trace-propagation-targets" /> is not provided, trace data is attached to every outgoing request from the instrumented client.
 
 </ConfigKey>
+
+<ConfigKey name="enable-tracing-when-crashes">
```

Comment:
This is a very odd name in light of us removing `enableTracing` from the SDKs.

Reply:
Thanks for pointing that out. Naming is hard 😅. It's basically finishing transactions bounded to the scope when an app crashes. The code is here https://github.com/getsentry/sentry-cocoa/pull/4504. 

I also considered `enableCrashTracing` and `enableTracingCrashes` but that's closer to tracing actual crashes, which the SDK already does. And I have a typo there. It should be `enableTracingWhenCrashing`. Is it still odd?

Reply:
@philipphofmann do we need a flag for this actually? That's the default behavior on android. Does it introduce any unwanted side-effects?

Reply:
> Does it introduce any unwanted side-effects?

Maybe yes. Other signal handlers might be effected, and the app can get stuck after writing the crash report to disk. So my plan is to make this the default behaviour, but for now it should be an experimental feature.

> Ok, so this will only work if tracing is actually enabled, and will then persist the transaction to disc?

Yes, indeed.

Reply:
Ok, but then it does not enable tracing. So I would suggest we call it something less confusing, like `persistTracesOnDiscIfCrashed`.

Reply:
We prefix everything with enable in the Cocoa SDK, so I would like to keep that pattern. What about `enablePersistingTracesWhenCrashing`, @cleptric?

Reply:
Also ok